go/engine/device_add.go

```diff
@@ -113,7 +113,7 @@ func (e *DeviceAdd) Run(m libkb.MetaContext) (err error) {
 
 	// make a new secret; continue to generate legacy Kex2 secrets for now.
 	kex2SecretTyp := libkb.Kex2SecretTypeV1Desktop
-	if provisioneeType == keybase1.DeviceType_MOBILE || m.G().GetAppType() == libkb.DeviceTypeMobile {
+	if provisioneeType == keybase1.DeviceType_MOBILE || m.G().GetAppType() == libkb.MobileAppType {
```

Comment:
lol this seems like it was just wrong before and only happened to work

Reply:
yup. constants happened to have the same value.